xbmc/addons/AddonInstaller.cpp

```diff
@@ -423,6 +423,11 @@ bool CAddonInstaller::InstallFromZip(const std::string &path)
       eventLog->AddWithNotification(EventPtr(
           new CNotificationEvent(24045, StringUtils::Format(g_localizeStrings.Get(24143), path),
                                  "special://xbmc/media/icon256x256.png", EventLevel::Error)));
+
+    CLog::Log(
```

Comment:
looks like you can get here if `CDirectory::GetDirectory(zipDir, items, "", DIR_FLAG_DEFAULTS)` fails but that is not being part of the log line

Reply:
Yep, but if isfolder is true and item size is 1 you know that this part is at fault.

Reply:
Additionally, `CDirectory::GetDirectory` should already log if it fails to do it's job, so I think my log is sufficient.